addons/mail/models/models.py

```diff
@@ -31,3 +34,123 @@ def _notify_email_header_dict(self):
         return {
             'X-Odoo-Objects': "%s-%s" % (self._name, self.id),
         }
+
+    @api.model
+    def _notify_admins(self, subject, body, doc=False, **kwargs):
+        """
+            Posts notifications about system failures on Admins channel (recreates the channel if it no longer exists).
+        """
+        ir_model_data = self.env['ir.model.data']
+        model_mail_channel = self.env['mail.channel']
+        odoobot_id = ir_model_data.xmlid_to_res_id("base.partner_root")
+        # Get channel
+        admins_channel = ir_model_data.xmlid_to_object('mail.channel_admins') or model_mail_channel.sudo().search([('name', '=', str('Administrators'))])
```

Comment:
str('Administrators') is useless, as this is already a string. 

But on the other hand, the field name is translatable. This search won't work if:
- You're not in English
- The channel has been renamed. 

Usually searching on the name is not a good idea. 

So please define the channel as data in a xml file + prevent the unlink.